main.nf

```diff
@@ -1036,42 +1036,232 @@ pairBam = pairBam.dump(tag:'BAM Somatic Pair')
 intPairBam = pairBam.spread(bedIntervals)
 bamMpileup = bamMpileup.spread(intMpileup)
 
-// MuTect2, FreeBayes
-(pairBamMuTect2, pairBamFreeBayes) = intPairBam.into(3)
+// Mutect2, FreeBayes
+(pairBamMutect2, pairBamFreeBayes) = intPairBam.into(3)
 
 // STEP GATK MUTECT2
 
 process Mutect2 {
     tag {idSampleTumor + "_vs_" + idSampleNormal + "-" + intervalBed.baseName}
 
     input:
-        set idPatient, idSampleNormal, file(bamNormal), file(baiNormal), idSampleTumor, file(bamTumor), file(baiTumor), file(intervalBed) from pairBamMuTect2
-        set file(genomeFile), file(genomeIndex), file(genomeDict), file(dbsnp), file(dbsnpIndex) from Channel.value([
+        set idPatient, idSampleNormal, file(bamNormal), file(baiNormal), idSampleTumor, file(bamTumor), file(baiTumor), file(intervalBed) from pairBamMutect2
+        set file(genomeFile), file(genomeIndex), file(genomeDict), file(intervals), file(commonSNPs), file(commonSNPsIndex) from Channel.value([
```

Comment:
replace `commonSNPs` with `germline_ressource` to be consistent with GATK parameters

Reply:
Need to update these as well:
https://github.com/nf-core/sarek/blob/ca2a0b0e41128f86d24ab683074592ca26b81df8/conf/igenomes.config#L16-L17
https://github.com/nf-core/sarek/blob/ca2a0b0e41128f86d24ab683074592ca26b81df8/conf/igenomes.config#L31-L32
https://github.com/nf-core/sarek/blob/ca2a0b0e41128f86d24ab683074592ca26b81df8/conf/genomes.config#L16-L17
https://github.com/nf-core/sarek/blob/ca2a0b0e41128f86d24ab683074592ca26b81df8/conf/genomes.config#L31-L32
https://github.com/nf-core/sarek/blob/ca2a0b0e41128f86d24ab683074592ca26b81df8/conf/genomes.config#L46-L47